Dima/Classic_Algorithms/closest_pair_problem/closest_pair_problem.py

```diff
@@ -0,0 +1,50 @@
+import math
+from Dima.Decorators.time_decorator import timer
+
+
+def create_route_matrix(dot_list):
+    try:
+        route_matrix = [[0] * len(dot_list) for index in range(len(dot_list))]
+        for index_row in range(len(dot_list)):
+            for index_col in range(index_row+1, len(dot_list)):
+                try:
+                    route_matrix[index_row][index_col] = ((dot_list[index_row][0] - dot_list[index_col][0]) ** 2 +
+                                                          (dot_list[index_row][1] - dot_list[index_col][1]) ** 2) ** (1 / 2)
+                except TypeError:
+                    raise TypeError('TypeError')
```

Comment:
Please separate exceptions 
for example first raise should be TypeError_1
Second raise TypeError_2
And add tests